src/material/core/theming/tests/theming-mixin-api.spec.ts

```diff
@@ -17,7 +17,9 @@ const localPackageSassImporter = createLocalAngularPackageImporter(packagesDir);
 const mdcSassImporter = {
   findFileUrl: (url: string) => {
     if (url.toString().startsWith('@material')) {
-      return pathToFileURL(path.join(runfiles.resolveWorkspaceRelative('./node_modules'), url));
+      return pathToFileURL(
+        path.join(runfiles.resolveWorkspaceRelative('./node_modules'), url),
+      ) as URL;
```

Comment:
That seems surprising, `pathToFileURL` should return an `URL` IIRC

Reply:
It returns its own URL type that used to overlap with the built-in one, but it doesn't anymore.